### PR TITLE
Road to 1.0.0

### LINKED ISSUE #6: Use custom style
Add the ability to send to button custom style/class name.

### PR DESCRIPTION
**Pull request by alepop**
In this PR I'm providing some changes for `lisk-buttons`:

- Remove `spectre.css` styles and `shadow dom` for buttons. Now all rendered buttons are wrapped with HTML element with default class name but without any styles. An end user can change this default class names.
- Remove tooltip and loading spiner.
- Now `onError` callback will redirect a user to the https://lisk.io/hub to download Lisk Hub (implement logic for Android/IOS market redirection too)
- Add `tslint` linter based on `stencil.js` guideline 

close #6

### CODE REVIEW

File: src/components/lisk-button-vote/lisk-button-vote.tsx [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
 
-  open() {
+  private open = (): void => {
     const { votes, unvotes } = this;
     const url = getURL({ votes, unvotes, kind: 'vote' });
     this.openUrl(url);
   }
 
-  getDefaultTitle() {
+  private getTitel = () => this.buttonTitle || this.getDefaultTitle();
```
Comment (hirishh) [inline src/components/lisk-button-vote/lisk-button-vote.tsx]: typo here `getTitel`

File: src/components/utils/index.ts [hunk 2]
```diff
@@ -11,29 +11,52 @@ interface SendLinkParams {
...
+};
+
+export interface AppUrls {
+  ios?: string;
+  android?: string;
+  desktop: string;
+}
+
+export const appLinks: AppUrls = {
+  desktop: 'https://www,lisk.io/hub',
```
Comment (hirishh) [inline src/components/utils/index.ts]: typo there is a comma in the url

**Review by hirishh — CHANGES_REQUESTED** [review]
just 2 typos. LGTM

**Comment by alepop** [comment]
@hirishh fixed, thanks.

**Review by hirishh — APPROVED** [review]
LGTM

File: package.json [hunk 3]
```diff
@@ -20,16 +20,16 @@
   },
```
Comment (hendrikhofstadt) [inline package.json:21]: We should maybe add the tslint checks to the CI tests so no code that doesn't pass the linter gets pushed ?

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]
I like the idea of decoupling the design.
Looks good to me

File: package.json (hunk 3 above)
Comment (alepop) [reply package.json:21]: @Slamper done 👍 thanks for your advice.

**Review by hendrikhofstadt — APPROVED** [review]

**Pull request merged**